tests/subsys/bootloader/b0_ram_cleanup/sysbuild/nrf54l15dk_nrf54l15_cpuapp_common.dtsi

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	cpuapp_sram@2003FFF0 {
```

Comment:
why is this in the sysbuild folder? What does this file have to do with sysbuild?